Verse/src/Schemas/JSON/JSONValue.cs

```diff
@@ -16,7 +15,12 @@ namespace Verse.Schemas.JSON
 		/// <summary>
 		/// Numeric value (only set if type is number).
 		/// </summary>
-		public readonly double Number;
+		public double Number => this.numberMantissa * this.number10PowExponent;
+
+		public decimal NumberAsDecimal => this.numberMantissa * (decimal) this.number10PowExponent;

```

Comment:
As this class is public, could you please insert a comment to explain what's the benefit of this property?

Reply:
done